src/helics/shared_api_library/helicsExport.cpp

```diff
@@ -979,6 +979,15 @@ void helicsQuerySetQueryString(helics_query query, const char* queryString, heli
     queryObj->query = AS_STRING(queryString);
 }
 
+void helicsQuerySetOrdering(helics_query query, int32_t mode, helics_error* err)
```

Comment:
For the mode would we want to use the enum typedef instead of `int32_t`?

Reply:
I am not sure I want enum in the definitions for the C interface.  That can lead to type difficulties.